adala/runtimes/_litellm.py

```diff
@@ -729,8 +751,9 @@ class AsyncLiteLLMVisionRuntime(AsyncLiteLLMChatRuntime):
 
     def init_runtime(self) -> "Runtime":
         super().init_runtime()
-        if not litellm.supports_vision(self.model):
-            raise ValueError(f"Model {self.model} does not support vision")
+        # model_name = self.model
```

Comment:
can we leave a comment explaining why we have to do this? And also, ideally, reminding us that we can bring it back after the next release of litellm? (Since the vertex models have the appropriate supports_vision attr now)

Reply:
Also, maybe we could at least still run this for non-vertex models?

Reply:
updated